apps/coscrad-frontend/src/components/higher-order-components/global-search.tsx

```diff
@@ -0,0 +1,62 @@
+import { FormFieldType, ResourceCompositeIdentifier, ResourceType } from '@coscrad/api-interfaces';
+import { Box, MenuItem, Select } from '@mui/material';
+import { useState } from 'react';
+import { DynamicSelect } from '../dynamic-forms/dynamic-form-elements/dynamic-select';
+import { fullViewCategorizablePresenterFactory } from '../resources/factories/full-view-categorizable-presenter-factory';
+import { AggregateDetailContainer } from './aggregate-detail-container';
+
+interface GlobalSearchProps {
+    onNewSelection?: (resourceCompositeIdentifier: ResourceCompositeIdentifier) => void;
+}
+
+export const GlobalSearch = ({ onNewSelection }: GlobalSearchProps) => {
```

Comment:
Good to see this

Reply:
Yea. It's a bit of a stretch to call it full global search, but it's a nice placeholder. Can't wait to really get into full-text search.